nixos/modules/installer/tools/nixos-version.sh

```diff
@@ -13,6 +13,13 @@ case "$1" in
     fi
     echo "@revision@"
     ;;
+  --configuration-revision)
+    if [[ "@configurationRevision@" =~ "@" ]]; then
+      echo "$0: configuration revision is unknown" >&2
```

Comment:
Could you fix the other errors to go to stderr as well?

Reply:
Yes - I wasn't sure if this was in the scope of this PR, but I think it makes sense :)